lib/CompletionExtension.php

```diff
@@ -81,9 +83,23 @@ private function registerCompletion(ContainerBuilder $container): void
             return new TypedCompletorRegistry($mapped);
         });
 
-        $container->register(self::SERVICE_FORMATTER, function (Container $container) {
+        $container->register(self::SERVICE_SHORT_DESC_FORMATTER, function (Container $container) {
```

Comment:
This is a BC break - might be used by another (phpactor) package (hover uses it in language-server-extension f.e.)

Reply:
I think I covered all cases, if you prefer I can revert this change.
It was awkward to me to have previous name but it's possible.

Reply:
As seen in slack I resolve this because all occurrences of `SERVICE_FORMATTER` and `TAG_FORMATTER` were properly renamed.